src/Http/Http.Results/src/Microsoft.AspNetCore.Http.Results.csproj

```diff
@@ -28,4 +33,16 @@
     <InternalsVisibleTo Include="Microsoft.AspNetCore.Http.Results.Tests" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="ResultsCache.StatusCodes.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>ResultsCache.StatusCodes.cs</LastGenOutput>
+    </None>
+  </ItemGroup>
+
+  <ItemGroup>
+    <!-- This is the T4 template service and is added by VS anytime you modify a T4 template. Required for .tt files. -->
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
```

Comment:
ew, but OK 😄

Reply:
I don't like it too, and I can actually remove but it will come back, and the next dev will need to remove it again. If that is fine i would prefer to remove it, including the `Compile`/`None` items added by VS.